docs/v4/mini-faq.md

```diff
@@ -0,0 +1,198 @@
+# Mini-FAQ
+
+@jesus-seijas-sp collected a mini FAQ for some v4 topics:
```

Comment:
I have started a new branch to create updated docs (docs/improve-docs). This PR will be merged before, then we'll  adapt this content to there, thanks!

Reply:
by the way, next time it would be better to split content like this into 2 PRs (documents on one side and new features on the other, except if they are tightly linked). That would accelerate having merged the updated docs

Reply:
I know ... that PR piled up interestingly :-) Normally not really my goal